ironfish/src/sdk.ts

```diff
@@ -220,6 +220,7 @@ export class IronfishSdk {
 
     if (this.config.get('enableRpcTcp')) {
       const namespaces = [
+        ApiNamespace.account,
```

Comment:
This is breaking security for RPC based calls to the TCP adapter. If you want this you should set `rpcTcpSecure` flag to true. You can also use the global flag `--rpc.tcp.secure`. Look at line 236 below in your own PR. https://github.com/iron-fish/ironfish/pull/1709/files#diff-c6255a87960ceb1747bb6bfadb135b4219244345cc96dc33c760fbad5c2122dcR237. You can also use the flag `

Reply:
Disagree. Transaction based routes are supported for tcp adapter, so real funds can be sent to other external account. So it's not a problem for account based routes. By the way, I have opened an FIP for Ironfish permission control system and work on it now. So this security problem would be fixed with auth instead of route control. [Auth System FIP](https://github.com/iron-fish/ironfish/issues/1677)